src/main/java/com/dreammaster/main/MainRegistry.java

```diff
@@ -151,6 +151,20 @@ public static void AddLoginError(String pMessage) {
         }
     }
 
+    /**
+     * Returns true on a client
+     */
+    public static boolean isClient() {
+        return proxy instanceof ClientProxy;
```

Comment:
Why not put this in the proxy class like GT did? I think this could cause problems since ClientProxy implements an interface marked as client only

Reply:
that's possible, I will change it